starport/pkg/localfs/search_test.go

```diff
@@ -0,0 +1,115 @@
+package localfs
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupGlobTests(t *testing.T, files []string) string {
+	t.Helper()
+	tmpdir, err := os.MkdirTemp("", "glob-test")
+	require.NoError(t, err)
+
+	t.Cleanup(func() {
+		os.RemoveAll(tmpdir)
+	})
+	for _, file := range files {
+		fileDir := filepath.Dir(file)
+		fileDir = filepath.Join(tmpdir, fileDir)
+		if _, err := os.Stat(fileDir); err != nil && os.IsNotExist(err) {
```

Comment:
```suggestion
```

In-fact we can just delete this and MkdirAll anyway because MkdirAll already makes this check.